modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -49,7 +50,8 @@ will only run on nodes that have the `br0` bridge connected.
 the network for this NetworkAttachmentDefinition. Do not change this field unless
 you want to use a different CNI.
 <4> You must substitute the actual name of the bridge, if it is not `br0`.
-<5> Required. This allows the MAC pool manager to assign a unique MAC address to the connection.
+<5> Optional: The VLAN tag.
```

Comment:
With my context, this doesn't look like enough information. If your users are going to know what it means, it's fine, but I think it needs another half-sentence.

Reply:
Yeah, the vlan is established outside of the cluster. A user will already about it in order to use it.